src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Let's follow the ducks pattern for these action type values. For example, the value of ADD_BOOK should be `bookstore/books/ADD_BOOK`. Kindly fix them.